pages/1.11/release-notes/1.11.0-rc4/index.md

```diff
@@ -89,32 +89,30 @@ Please try out the new features and updated data services. Provide any feedback
 - Upgrades from 1.10.5 to 1.11.0-rc4 are supported, but not recommended.
 - Upgrades from 1.11.0-rc1 to 1.11.0-rc4 are _not supported_ in 1.11 Release Candidate 4.
 
-# <a name="fixed-issues"></a>Improvements and Major Issues Fixed since 1.11.0 Release Candidate 1
-- DCOS-19500  - Fix cosmos/service/update failure when Marathon is deploying
+# <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0 Release Candidate 4
+- COPS-2201 - Fix `dcos-diagnostics` crashing after running gen_resolvconf.py.
+- COPS-2465 - License reporting is inconsistent. [enterprise type="inline" size="small" /]
+- DCOS-19500 - Fix cosmos/service/update failure when Marathon is deploying.
+- DCOS-20064 - Verify Azure and AWS advanced templates work with the licensing parameter. [enterprise type="inline" size="small" /]
```

Comment:
How about changing this to `Support License Parameters in Azure and AWS templates`